benchmarks/distributed/rpc/parameter_server/configurations/benchmark_configurations.json

```diff
@@ -4,5 +4,17 @@
         "ps_count": 0,
         "rpc_async_timeout": 15,
         "batch_size": 5
+    },
+    "4": {
+        "trainer_count": 2,
+        "ps_count": 1,
+        "rpc_async_timeout": 30,
+        "batch_size": 10
+    },
+    "5": {
```

Comment:
I removed this file in the updates.

Reply:
I removed this file. all these arguments in the configurations have been moved to the launcher arguments. I added a help section to each argument in case the name is not self explanatory.